src/kernel/src/syscalls/mod.rs

```diff
@@ -134,6 +135,26 @@ impl Syscalls {
         Ok(self.vp.id().into())
     }
 
+    unsafe fn setsid(&self) -> Result<Output, Error> {
+        // Check if current thread has privilege.
+        VThread::current().priv_check(Privilege::SCE680)?;
+
+        // Check if the process already become a group leader.
+        let mut group = self.vp.group_mut();
+
+        if group.is_some() {
+            return Err(Error::Raw(EPERM));
+        }
+
+        // Set the process to be a group leader.
+        let id = self.vp.id();
+
+        *group = Some(VProcGroup::new(id));
+        info!("The virtual process become a group leader.");
```

Comment:
Change the info to be more concise and properly written.

```suggestion
        info!("Virtual process now set as group leader.");
```